### PR TITLE
Update org

### PR DESCRIPTION
**Pull request by makandre**
Enhance the register org flow to allow for update

### CODE REVIEW

File: actions/register_org.py [hunk 1]
```diff
@@ -21,16 +22,29 @@ def run(self, user, url, token, github_type, repositories, event_type_whitelist)
             dict={}
         
         user = user.strip()
-        url = url.strip()
-
+        url = (url or '').strip()
+
```
Comment (akihikokuroda) [inline actions/register_org.py]: Extra chars.

File: actions/register_org.py [hunk 2]
```diff
@@ -21,16 +22,29 @@ def run(self, user, url, token, github_type, repositories, event_type_whitelist)
...
+            org['repositories'] = list(set(org['repositories']).union(repositories))
+            org['event_type_whitelist'] = list(set(org['event_type_whitelist']).union(event_type_whitelist))
+
+        dict[key]=org
         gitorgs=json.dumps(dict)
 
         client.keys.update(KeyValuePair(name='git-orgs', value=gitorgs, secret=True))
 
-        return list(dict.keys())
+        return list(filter_orgs(dict))
```
Comment (akihikokuroda) [inline actions/register_org.py]: I think that it's better to return the new or updated entry only.

File: actions/delete_registered_org.py [hunk 3]
```diff
@@ -25,4 +26,4 @@ def run(self, user, url):
 
         client.keys.update(KeyValuePair(name='git-orgs', value=gitorgs, secret=True))
 
-        return list(dict.keys())
+        return list(filter_orgs(dict))
```
Comment (akihikokuroda) [inline actions/delete_registered_org.py]: I think that it's better to return the deleted entry only.

**Review by akihikokuroda — CHANGES_REQUESTED** [review]
Looks good.  Thanks.  I have comments about the return values.

**Comment by makandre** [comment]
I've made the suggested changes

**Review by akihikokuroda — APPROVED** [review]
Andrew, looks good.  Thanks.

**Pull request merged**